### PR TITLE
DOC Update README.md for CMS 5

### PR DESCRIPTION
**Pull request by GuySartorelli**
## Issue

- https://github.com/silverstripe/.github/issues/20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,21 +4,11 @@
...
-* [jdorn/sql-formatter](https://github.com/jdorn/sql-formatter)
-
 ## Installation
 
-You can install the debug bar with Composer:
-
+```sh
+composer require lekoala/silverstripe-debugbar
 ```
-composer require --dev lekoala/silverstripe-debugbar
```
Comment (sabina-talipova) [inline README.md:18]: Small question. Why do we recommend to install as general dependency, not dev dependency?

Comment (ChloeSartorelli) [reply README.md:18]: Good catch. This was an automated change, so I didn't see this. I'll add back the `--dev` part.

Comment (sabina-talipova) [inline README.md]: This difference.

Comment (ChloeSartorelli) [reply README.md:18]: Done

**Review by sabina-talipova — APPROVED** [review]
LGFM

**Pull request merged**